src/backend/core/api/viewsets/entitlements.py

```
@@ -94,10 +96,49 @@ def get(self, request):
+                        "account_id": organization.id,
+                    },
+                )
+
+            # Resolve entitlements.
+            for entitlement_type in entitlements_by_type:
+                entitlements_of_type = entitlements_by_type[entitlement_type]                
                 entitlement_data = resolver.resolve(
-                    {**entitlement_context, "entitlement": entitlement}
+                    {**entitlement_context, "entitlements": entitlements_of_type}
                 )
                 entitlements_data = {**entitlements_data, **entitlement_data}
```

Ruff (0.14.8)

Extracting value from dictionary without calling `.items()`

(PLC0206)